turbine/src/retransmit_stage.rs

```diff
@@ -84,55 +83,6 @@ struct RetransmitStats {
     unknown_shred_slot_leader: usize,
 }
 
-impl RetransmitStats {
-    fn maybe_submit(
```

Comment:
any reason to move this part of the code?
can this also be kept same place as before?

Reply:
sure but then we end up with 2 impl blocks for the same struct. Not sure which option is better. Also git blame -C will typically correctly see right through such moves.

Reply:
I can see people prefer one style versus the other and neither is wrong.
But if we want to keep making changes from one style to the other we keep churning the code because people have different style preferences.

Reply:
you are right, lets keep it where it is.